### ISSUE TITLE
Refactor ucrSave

### ISSUE THREAD

**Issue by lloyddewit** [IDEMSInternational/R-Instat#5688]
`ucrSave` is a widely used user control. It provides a label, text box and combo box. They are displayed in a row. There are no known bugs in `ucrSave` but the code looks unstructured and fragile. If the control is used in a new way then bugs could potentially be exposed.

The class is self-contained. No knowledge of the R-Instat architecture is needed to understand the class's code. Therefore it would be a good candidate to be one of the first classes to refactor.

**Comment by lilyclements** [comment]
@lloyddewit has this been done?

**Comment by lloyddewit** [comment]
> [@lloyddewit](https://github.com/lloyddewit) has this been done?

@lilyclements No, I don't think so. However, I don't think this issue will ever be high enough priority because it's more likely that effort will be invested in replacing these classes instead. So I suggest to close this issue.
